### ISSUE TITLE
Script installs php5 but requires php7

### ISSUE THREAD

**Issue by boppreh** [PocketMine/PocketMine-MP#4165]
https://www.pocketmine.net/ -> Get PocketMine-MP -> Linux -> Raspberry Pi / ODROID

> Navigate where you want to install/update PocketMine-MP.
> 
> Run the following code. It will download PocketMine-MP, download the PHP binaries or compile it if binaries are not available.
> 
> wget -q -O - http://get.pocketmine.net/ | bash
> 
> Run ./start.sh, PocketMine-MP will now start.

```
~/minecraft> wget -q -O - http://get.pocketmine.net/ | bash
[*] Found PocketMine-MP Final_1.4.1 (build 980) using API 1.11.0
[*] This stable build was released on Tue 17 Mar 17:00:47 UTC 2015
[+] The build was signed, will check signature
[*] Installing/updating PocketMine-MP on directory ./
[1/3] Cleaning...
[2/3] Downloading PocketMine-MP Final_1.4.1 phar... done!
[*] Checking signature of PocketMine-MP.phar
gpg: Signature made Tue 17 Mar 2015 17:16:17 UTC using RSA key ID A7837F89
gpg: Good signature from "Shoghi Cervantes <shoghicp@gmail.com>"
gpg:                 aka "keybase.io/shoghicp <shoghicp@keybase.io>"
[+] Signature valid and checked!
[3/3] Obtaining PHP: detecting if build is available...
[3/3] Raspberry Pi PHP build available, downloading PHP_5.6.10_ARM_Raspbian_hard.tar.gz... checking... regenerating php.ini... done
[*] Everything done! Run ./start.sh to start PocketMine-MP

~/minecraft> ./start.sh
Couldn't find a working PHP 7 binary, please use the installer.

~/minecraft> ls bin/
php5
```

Using Raspberry Pi 2. Beta and development versions have the same problem.

I tried compiling PHP 7 with the required flags. Server starts, users connect (Windows 10 Beta client), but inventory behaves oddly (#4028) with server errror

> Notice: Undefined offset: 0 in phar:///media/king/minecraft/temp/PocketMine-MP.phar/src/pocketmine/Player__32bit.php on line 2581
> [03:06:40] [RakLibServer thread/NOTICE]: Blocked 10.0.0.105 for 5 seconds

Tried changing `./start.sh` to use the downloaded PHP 5 instead. Server starts, but users fail to connect ("Locating server...") and then 

> [03:42:54] [Server thread/INFO]: [/10.0.0.110:19132] logged out due to Login timeout

I've been trying to get this to work for over five hours. What is wrong here?

**Comment by robske110** [comment]
https://bintray.com/pocketmine/PocketMine/Unix-PHP-Binaries/view

**Comment by boppreh** [comment]
The only Raspbian version there is `PHP_5.6.10_ARM_Raspbian_hard.tar.gz`

Tried [`PHP_7.0.3_x86_Linux.tar.gz`](https://bintray.com/pocketmine/PocketMine/download_file?file_path=PHP_7.0.2_x86_Linux.tar.gz) but it gives

```
~/minecraft> ./start.sh
./start.sh: line 54: /home/boppreh/minecraft/bin/php7/bin/php: cannot execute binary file: Exec format error
./start.sh: line 54: /home/boppreh/minecraft/bin/php7/bin/php: Success
```

Even if that worked, the setup script is still incorrect, no?

**Comment by coben07** [comment]
The installer script is still broken, installing php 5 but pocketmine requires php7.   I tried installing php 7 directly as robske110 suggested, but it still doesn't pick it up.  Can we just get the installer script updated?    Short of that, can we get better instructions to get it to recognize php 7?   Thanks for your help.

**Comment by coben07** [comment]
Looks like they do have an updated installer script in github, just not published to the website.  It uses php 7 and can be downloaded directly with the following link until they get it published to the site.
https://github.com/PocketMine/php-build-scripts/blob/master/installer.sh

**Comment by Attroxhippo** [comment]
Try renaming the file. From Php 5 to 'php 7'

**Comment by Attroxhippo** [comment]
It should work. The script just accidentally names it wrong

**Comment by PrimusLV** [comment]
Wanna test https://github.com/PrimusLV/PM-Installer ?

**Comment by boppreh** [comment]
I got it working using the precompiled [PHP7 for Raspberry Pi](https://drive.google.com/file/d/0B60O0ROCfeCmVW1idWx1WFY5RXM/view?usp=sharing) from [this thread in the forums](https://forums.pocketmine.net/threads/php7-with-pthreads-for-raspberry-pi.15424/). It functions as a workaround, but the installation script on the home page is still incorrect.

Also Windows 10 crafting is still broken (#4028) and I just discovered that PocketMine [doesn't support mobs and fire](https://gaming.stackexchange.com/questions/214433/mobs-arent-spawning-in-my-minecraft-pe-server). Unfortunately I was looking for a survival server and this is too limited.

Sorry @PrimusLV , didn't get to test your script.
